CRM/Report/Form/Pledge/Summary.php

```diff
@@ -268,25 +268,25 @@ public function groupBy() {
                 )) {
                   $append = '';
                 }
-                $this->_groupBy[] = "$append {$this->_params['group_bys_freq'][$fieldName]}({$field['dbAlias']})";
```

Comment:
Hmm - If I'm going to test out these reports I'd rather test the 'right' change.  I just tested the Walklist report - with the  existing (non 7.1 compliant) order by function sort by ascending works, sort by descending does not work, section break does not work. Removing the order by function causes it to work correctly (and fixes your php problem) - note the walklist report needs to be enabled under manage templates.